### ISSUE TITLE
ice hangs on compile error

### ISSUE THREAD

**Issue by dastrobu** [jakeheis/Ice#16]
There are situations, where ice hangs forever on certain compile errors. This can be easily reproduced in the following way (ice-0.5.0, Swift-4.1, macOS-10.13.4):

```
mkdir foo
cd foo
ice init 
ice build
cd ..
mv foo bar
cd bar
ice build
```
on the last command, ice never exits and does not show any error. When invoking the build via Swift PM directly, one gets:
```
$ swift build
Compile Swift Module 'foo' (1 sources)
<unknown>:0: error: PCH was compiled with module cache path '...', but the path is currently '...'
<unknown>:0: error: missing required module 'SwiftShims'
error: terminated(1): /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build-tool -f .../bar/.build/debug.yaml main output:

```
Note, I do not expect that ice compiles the sources, but I would expect that ice shows the same errors as the build from the Swift PM.

**Comment by jakeheis** [comment]
Thanks for the report and easy instructions to reproduce! Should be fixed by 3fc27f4fc32d7a81d87f146aedec6d1746e43dd7, will close once a new release is tagged.

**Comment by dastrobu** [comment]
Great! Would be nice to see the Homebrew formula updated as well at some time.
